nixos/modules/misc/nixpkgs.nix

```diff
@@ -54,6 +55,12 @@ let
     check = builtins.isAttrs;
   };
 
+  defaultPkgs = import ../../.. {
```

Comment:
As a side note and while at it: Maybe it would make sense to import `../../../pkgs/top-level` instead, so it won't go via `impure.nix`.